modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerContainerFragment.kt

```diff
@@ -211,8 +215,10 @@ class PlayerContainerFragment : BaseFragment(), HasBackstack {
     }
 
     fun openUpNext() {
-        val upNextFragment = UpNextFragment.newInstance(source = UpNextSource.PLAYER)
-        (activity as? FragmentHostListener)?.showBottomSheet(upNextFragment)
```

Comment:
This change was made because as far as I can tell there's no point in creating a new `UpNextFragment` when one is already added to `childFragmentManager` in this fragment. This change speeds up showing the queue when tapping on the top right button as there we just show what already exists.